src/System.Net.Requests/tests/FileWebRequestTest.cs

```diff
@@ -232,7 +229,7 @@ public abstract class FileWebRequestTestBase
 
     public abstract class AsyncFileWebRequestTestBase : FileWebRequestTestBase
     {
-        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "dotnet/corefx #17842")]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "concurrent read/write not supported on netfx")]
```

Comment:
I assume this is referring to the fix made in https://github.com/dotnet/corefx/pull/12231.  If so, might want to include the PR link or number.